### ISSUE TITLE
Updates

### ISSUE THREAD

**Issue by bcmux** [miller-productions/leanmeanbook#1]
Hey Dan!

Your eBook has by far been a great platform at launching me into the MEAN stack. I really appreciate how easy you made it to follow along the entire time. I wasn't really sure how to get in touch with you other than through GitHub, and I do have a few questions.

Throughout the book, there were brief discussions about setting up configs and env variables specifically for other environments such as Test and Production. While the book briefly went into the testing stages of the app, I was really curious on your take on the best practices to take an app into production. I absolutely loved your approach to everything else in the book, and it made me that much eager to get to that section, but alas it was never there. :(

I've gone as far as implementing Angular 5 onto my app, so I would love to hear back from you on what you typically would do when getting a MEAN stack app production ready (ie; How do you typically build your dist files?).

Thanks again for the awesome read, Dan!

**Comment by miller-productions** [comment]
Hi @bcmux 

Great to meet you and thanks for the feedback about the book. Sincerely appreciated.

I'm really glad you have gotten value from my work and that it has helped you launch into the MEAN stack. That was absolutely my goal - to help people get started in that space.

Apologies about the book lacking in the testing section, that is something I acknowledge. 

On that topic, and also regarding further information about build pipelines, etc... 

My recommendation is to take a look at the MEAN.JS project. 

See https://github.com/meanjs/mean and http://meanjs.org/

Lean MEAN Web App Machine is what you would call a 'starter pack' that launches you into being productive with the full capabilities of MEAN.JS.

All the material in the book is based around the MEAN.JS core, so anyone that has gone through the book should be quite familiar with MEAN.JS. It really is an awesome project (hence the reason I decided to write the book based on it) and the team involved there are very friendly and helpful.

Basically, MEAN.JS is a production ready framework complete with thorough build and testing capabilities (among many other features). It picks up where Lean MEAN Web App Machine leaves off.

I hope that helps.

Sincerely,
Dan

P.S. If you had a spare moment, it would mean a lot to me if you were able to drop a review into [the Amazon page for the book](https://www.amazon.com/dp/0648115402). Thanks in advance 😃
